content/hardware/07.opta/opta-family/opta/tutorials/17.opta-mqtt/content.md

```diff
@@ -0,0 +1,555 @@
+
+![CloudMQTT Instances](assets/opta-mqtt-tutorial-0005.png)
+
+To create a new instance, click on the green "+ Create New Instance" button at the top right of the page.
+
+You may need to update payment information before proceeding with instance creation.
+
+![New Instance on CloudMQTT without Credit Card](assets/opta-mqtt-tutorial-0006.png)
+
+After setting up a payment method, fill in the fields to customize the new instance:
+
+- **Name**: Set the name of the instance (e.g., **OptaInstance**).
+- **Plan**: Select the desired monthly plan (recommended: **Humble Hedgehog** at $5 per month).
+- **Tags**: Add any tags to identify the instance.
+
+![Creating a New Instance on CloudMQTT](assets/opta-mqtt-tutorial-0008.png)
+
+Select the server region for the instance and proceed by pressing the "Select region" button.
+
+For this tutorial, the "EU-West-1 (Ireland)" Data center region has been chosen.
+
+![Selecting Region for Instance on CloudMQTT](assets/opta-mqtt-tutorial-0009.png)
+
+Finally, click on "Review" and, if everything is correct, press "Create Instance".
+
+![Instance Review on CloudMQTT](assets/opta-mqtt-tutorial-0010.png)
+
+You'll be redirected to the list of instances, where you can find the necessary information to connect to your MQTT broker.
+
+![List of Instances on CloudMQTT](assets/opta-mqtt-tutorial-0011.png)
+
+Click on the name of the newly created instance (OptaInstance) to access the control dashboard.
+
+![Instance Dashboard on CloudMQTT](assets/opta-mqtt-tutorial-0012.png)
+
+All the information needed to connect Opta™ to your MQTT broker is displayed on the current page and can be copied and pasted into your sketch within the **arduino_secrets.h** file.
+
+- **Server** will be your MQTT host.
+- **User** is the username to connect to your MQTT broker.
+- **Password** is the password for the username.
+- **Port** is the MQTT port required for connection, usually just use the one indicated in the **Port** field.
+
+![Instance Info on CloudMQTT](assets/opta-mqtt-tutorial-0013.png)
+
+In the next chapter, you'll focus on learning what the sketch does, how to configure it correctly, and how to upload it to your Opta™.
+
+## Sketch
+
+### Overall Code Analysis
+
+You can download the complete example project from [here](assets/opta-mqtt-tutorial.zip).
+
+After downloading the .zip file, extract it and open the .ino file inside.
+
+![CloudMQTT Sketch](assets/opta-mqtt-tutorial-0014.png)
+
+Briefly analyzing the code, at the beginning of the .ino file, there are the necessary library inclusions.
+
+```cpp
+#include <ArduinoMqttClient.h>   // Include the ArduinoMqttClient library to handle MQTT connection
+#include <WiFi.h>                // Include the WiFi library to handle WiFi connection
```

Comment:
```suggestion
#include <WiFi.h>                // Include the WiFi library to handle Wi-Fi connection
```

Reply:
We should write the trademark Wi-Fi properly in code sections, but we don't need to use the `®` symbol here. I've update this in the places I found ⭐